### ISSUE TITLE
Downloaded files have informative names

### ISSUE THREAD

**Issue by ambrosejcarr** [chanzuckerberg/corpora-data-portal#75]
## Background

Historically, DCP/1 files had naming conventions that were hard to parse. We can leverage the metadata information that we have for each file to create high information content naming conventions that will help users understand file contents. 

## User Stories

_After a scientist downloads a file, they want to be able to easily determine its contents so that when they later return to it, they understand what it contains_

_When a scientist has downloaded multiple files in a short period of time, they need the names to contain enough information so that they can match the files to the downloads they requested_

## Acceptance Criteria
- [x] Specification for naming files
- [ ] Files served by browser have informative names

**Comment by ambrosejcarr** [comment]
@liaprins-czi I added a ticket to track your work on file naming (could you add the link in here?). If you think I missed any user stories, feel free to adjust.

**Comment by liaprins-czi** [comment]
Here is an in-progress document of the naming specs. So far it conveys the concept of the naming scheme, but I need to add more specifics next.
https://docs.google.com/document/d/1ovK8GvoSF5j7OC2-uIheljw7dTJErEXAQQsR36gEk6E/edit?usp=sharing

**Comment by liaprins-czi** [comment]
Here's where I've documented the naming scheme:
https://docs.google.com/document/d/1ovK8GvoSF5j7OC2-uIheljw7dTJErEXAQQsR36gEk6E/edit?usp=sharing

@ambrosejcarr Can you review and add any feedback as comments? Then I will incorporate.
Also can you let me know who else should review this (in addition to Calvin who I've tagged below)? Thanks!

@calvinnhieu I know we were going to sync on the naming scheme, can you take a look at what I've outlined in the link above? You can add any feedback as comments, or we can chat in person (or both). Thanks!

**Comment by calvinnhieu** [comment]
Not enough scientific bkg to comment on collisions/feasibility of rules, but from a technical standpoint all proposed abbreviations for tissue, assay and species are trivial enough to perform 👍. All for improving readability of filenames, but may not prioritize pushing this in for the 2/21 considering current bandwidth.

**Comment by ambrosejcarr** [comment]
Reviewed from a scientific perspective. @liaprins-czi this looks awesome. I left a bunch of questions and one gnarly metadata problem. 

@calvinnhieu I think there's enough complexity here that you should push forward with a convention that makes sense to you, and we can update your filenames at a later point.
